Qemu/QMP fails sporadically when running benchmarks

In case we run some benchmarks using Qemu hypervisor such as:
```
go test -bench=CreateStartStopDeletePodQemuHypervisorHyperstartAgentNetworkNoop -benchtime=60s
```
We get the following error:
```
WARN[0022] Unable to connect to unix socket (/tmp/virtcontainers-tmp-009901593/virtcontainers/pods/run/7f49d00d-1995-4156-8c79-5f5ab24ce138/monitor.sock): dial unix /tmp/virtcontainers-tmp-009901593/virtcontainers/pods/run/7f49d00d-1995-4156-8c79-5f5ab24ce138/monitor.sock: connect: no such file or directory 
ERRO[0022] Failed to connect to QEMU instance dial unix /tmp/virtcontainers-tmp-009901593/virtcontainers/pods/run/7f49d00d-1995-4156-8c79-5f5ab24ce138/monitor.sock: connect: no such file or directory 
--- FAIL: BenchmarkCreateStartStopDeletePodQemuHypervisorHyperstartAgentNetworkNoop-4
        api_test.go:1393: Could not create and start pod: Did not receive the pod started notification
FAIL
```

I think this is caused by Qemu stopping very soon for unknown reasons. I have added a timeout before we call QMPStart() because I thought it was the monitor.sock creation that was taking some time, but obviously this is the opposite.
This issue will be solved by finding the reason why Qemu is stopping unexpectedly.

--- a/qemu.go
+++ b/qemu.go
@@ -25,6 +25,7 @@ import (
 	"runtime"
 	"strings"
 	"sync"
+	"time"
 
 	ciaoQemu "github.com/01org/ciao/qemu"
 	"github.com/01org/ciao/ssntp/uuid"
@@ -514,7 +515,19 @@ func (q *qemu) stopPod() error {
 		return err
 	}
 
-	return qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
+	if err := qmp.ExecuteQuit(q.qmpMonitorCh.ctx); err != nil {
+		return err
+	}
+
+	// Wait for the VM disconnection notification
+	select {
+	case <-q.qmpControlCh.disconnectCh:
+		break
+	case <-time.After(time.Second):
+		return fmt.Errorf("Did not receive the VM disconnection notification")
+	}
+
+	return nil
 }
 
 // addDevice will add extra devices to Qemu command line.

--- a/qemu.go
+++ b/qemu.go
@@ -206,6 +206,8 @@ func (q *qemu) appendNetworks(devices []ciaoQemu.Device, endpoints []Endpoint) [
 				ID:         fmt.Sprintf("network-%d", idx),
 				IFName:     endpoint.NetPair.TAPIface.Name,
 				MACAddress: endpoint.NetPair.VirtIface.HardAddr,
+				DownScript: "no",
+				Script:     "no",
 			},
 		)
 	}

